obspy/core/utcdatetime.py

```diff
@@ -981,63 +987,63 @@ def __unicode__(self):
         """
         return str(self.__str__())
 
+    def _operate(self, other, op_func):
+        if isinstance(other, UTCDateTime):
+            a = py3_round(self._ns, self.precision - 9)
+            b = py3_round(other._ns, self.precision - 9)
+        elif isinstance(other, (float, int)):
+            a = round(self.timestamp, self.precision)
+            b = round(float(other), self.__precision)
+        elif isinstance(other, datetime.datetime):
+            a = self.datetime
+            b = other
+        else:
+            return False
+        return op_func(a, b)
+
     def __eq__(self, other):
         """
         Rich comparison operator '=='.
 
         .. rubric: Example
 
-        Comparing two UTCDateTime object will always compare timestamps rounded
+        Comparing two UTCDateTime object will compare timestamps rounded
         to a precision of 6 digits by default.
```

Comment:
That phrasing is not really accurate as the default precision just happens to be 6. Might be a good opportunity to change it.